### ISSUE TITLE
FNF Mobile (by gamerbross) Super optimized

### ISSUE THREAD

**Issue by netonero** [luckydog7/Funkin-android#854]
https://www.mediafire.com/file/46k2kuit8h70qse/OptimizedFNFTEGarcUpd.apk/file
Changed:
Every possible changeable asset as a blank image (which means menuBGs, logobumpin, charactermenus, and week background and characters are gone)
Bugs:
Stress crashing

Unoptimized ver: #817

**Comment by arrivedercihehe** [comment]
a noble sacrifice

**Comment by AnthonyCTM** [comment]
> a noble sacrifice.
> at least it actually works! also what the fuck is up with the health system, missing one note Drains your health significantly

The note damage is at 1(neutral damage) so it's normal, also before in previous releases it haved a pretty lower damage so they increased it to make the game not that easy with the big recover that the notes give to you

**Comment by arrivedercihehe** [comment]
> > a noble sacrifice.
> > at least it actually works! also what the fuck is up with the health system, missing one note Drains your health significantly
> 
> The note damage is at 1(neutral damage) so it's normal, also before in previous releases it haved a pretty lower damage so they increased it to make the game not that easy with the big recover that the notes give to you

that would've worked IF the sustain notes actually gave you health, they don't give shit now

**Comment by AnthonyCTM** [comment]
> > > a noble sacrifice.
> > > at least it actually works! also what the fuck is up with the health system, missing one note Drains your health significantly
> > 
> > 
> > The note damage is at 1(neutral damage) so it's normal, also before in previous releases it haved a pretty lower damage so they increased it to make the game not that easy with the big recover that the notes give to you
> 
> that would've worked IF the sustain notes actually gave you health, they don't give shit now

Yeah that sucks but it's hard to code a week for a new platform with different settings and stuff

**Comment by hakerSALAD** [comment]
#1
